spec/controllers/validation_reports_controller_spec.rb

```diff
@@ -4,13 +4,12 @@
 
   routes { Krikri::Engine.routes }
 
-  describe 'GET #index' do
-    login_user
-
-    it 'redirects to /krikri/report_lists#index in absence of valid params' do
-      get :index
-      expect(response).to redirect_to('/krikri/report_lists')
-    end
-  end
-
+  # describe '#show' do
```

Comment:
Are these the specs that needed to get commented out because of the issues with Blacklight?